.github/workflows/ci-check.yml

```
@@ -45,3 +45,29 @@ jobs:
         with:
           name: recorded-videos
           path: cypress/videos/
+  python-tests:
```

```suggestion
  python-check:
```
Just because we're linting in addition to running tests